packages/fast-components-react-base/src/badge/badge.schema.json

```diff
@@ -4,11 +4,13 @@
     "description": "A badge component's schema definition.",
     "type": "object",
     "id": "@microsoft/fast-components-react-base/badge",
+    "formPluginId": "@microsoft/fast-components-react-base/badge",
```

Comment:
I’m assuming we need this to be different than the actual ID above, correct?

Reply:
It does not, as it is at root level. This merely adds a hook for the entire schema.